lib/puppet_forge/v3/base.rb

```diff
@@ -78,8 +78,8 @@ def where(params)
         end
 
         # Return a paginated collection of all modules
-        def all
-          where(nil)
+        def all(params = {})
+          where(params)
```

Comment:
On top of using .all to get the entire paginated collection, v1 of the puppet_forge gem allowed the use of .all just like where, this change makes that functionality still work in v2.